product_docs/docs/sql_assess/1/using.mdx

```diff
@@ -0,0 +1,260 @@
+---
+title: Using EDB SQL Assess
+navTitle: Using
+---
+
+The application SQL assessment tool (AssessSQL) provided by the EDB SQL Assess capability is a Java-based command line tool that the end user can download and install in their local environment. Once installed, it can assess SQL statements executed by the application that have been captured by an Oracle trace or extracted out of the Oracle database. The end user can run a test program or regression test suite that performs most of the possible user actions so that all queries get captured in the trace file. 
+
```

Comment:
@ccestes I believe that SQL Assess now provides the ability to assess plain SQL and SQL*Plus scripts that are not trace files or extracted into trace file format.  Assuming that is the case, some text about this should be provided here.  The text about running a test program or regression suite is still applicable to the trace file approach, but it would not be for plain/SQL*Plus SQL scripts.

@gabbasb can you confirm and offer assistance with coming up with the right text to add here to account for support of plain SQL scripts.

Reply:
Revised paragraph is as follows:
`The application SQL assessment tool (AssessSQL) provided by the EDB SQL Assess capability is a Java-based command line tool that the end user can download and install in their local environment. Once installed, it can assess SQL statements provided as an input file. The input file can either be Oracle trace file, extracted SQL file or plain SQL file. In order to create Oracle trace file, the end user can run a test program or regression test suite that performs most of the possible user actions so that all queries get captured in the trace file. To generates extracted SQL file the user can run edb_query_extractor SQL script.`